waterdata/templates/partials/footer.html

```diff
@@ -0,0 +1,10 @@
+<footer class="usa-footer usa-footer-slim" role="contentinfo">
+    <div class="usa-footer-secondary_section">
+        <div class="usa-grid">
+            <div class="usa-footer-logo">
+                <img class="usa-footer-slim-logo-img" src="{{ config.STATIC_ROOT }}/img/logo-img.png" alt="Logo image">
+                <h3 class="usa-footer-slim-logo-heading">U.S. Geological Survey</h3>
+            </div>
+        </div>
+    </div>
+</footer>
```

Comment:
There are actually some templates bundled with USWDS that we could possible import instead of making our own.